tests/integration/src/test/groovy/com/datadog/appsec/php/integration/CommonTests.groovy

```diff
@@ -46,13 +103,6 @@ trait CommonTests {
         assert trace.metrics._sampling_priority_v1 == 2.0d
 
         assert trace.metrics."_dd.appsec.waf.duration" > 0.0d
-        assert trace.metrics."_dd.appsec.event_rules.loaded" > 0.0d
```

Comment:
Rule diagnostics are only added to the first trace, but the order of the tests is not defined. If we can figure out how to define it we can add a test later on.